examples/different_integrators/oscilator_eval.m

```diff
@@ -0,0 +1,49 @@
+% BSD 2-Clause License
```

Comment:
we have now a lot of examples. what do you think about putting this into the oscialtor example folder - one file for fesd integrator, another for smoothed integrator.

Reply:
add matlab options for integrator to our integrator options class and add in the example example of chaning tolerances so that we can see that high accuracy can be reached also for lower values of sigma.
currently it saturates at 1e-3. see printed message from my last commit